### ISSUE TITLE
Decide how to handle updating friends list.

### ISSUE THREAD

**Issue by DevinHolland** [chaughawout/AP_Server#24]
Friends list will need to auto update in order to show online/offline friends appropriately. 

This can be done by requesting the friends list again every so often (10 seconds?)... however that would generate a lot of requests to the server.

Other way is push notifications: http://en.wikipedia.org/wiki/Push_technology. However, this may require having a dedicated server for them.

**Comment by chaughawout** [comment]
I don't think it will require a dedicated server for them, but it definitely scales better to design it that. We can use long polling for this which, you're right, does generate a lot of traffic.
